troubleshooting/debugging.md

```diff
@@ -98,6 +99,43 @@ Even if you see this message, you still have not lost logs yet. Since it will re
 
 When you see this message, you have lost logs. The other case that indicates log loss is when an input is paused, which is covered in the [overlimit error section](#overlimit-warnings).
 
+#### Common Network Errors
+
+First, please read the section [Network Connection Issues](#network-connection-issues), which explains that Fluent Bit's http client is very low level and logs errors that other clients may simply silently retry. That section also explains the `auto_retry_requests` option which now defaults to true. 
```

Comment:
[nit-pick]
 - `very` in "very low level" does not add additional details
 - `auto_retry_requests` (defaults to true) - let's treat this as the present